### DISCUSSION TITLE
Assigning IPv4 to Device

### DISCUSSION THREAD

**Discussion by nbecirovic** [netbox-community/netbox#13975] [Help Wanted!]
Does anyone know if IPv4 can be provisioned via CSV Data Import for Devices?  I don't see the field for it in CSV Field Options and I've tried setting static (/32) address for given Device through IPAM but it doesn't show up in the provisioned device.

**Comment by jcollie** [comment]
You don't assign IPs to devices, you assign IPs to interfaces by importing a CSV though the IP addresses page.
